charts/dev/theme-park/values.yaml

```diff
@@ -0,0 +1,36 @@
+image:
+  repository: ghcr.io/k8s-at-home/theme-park
+  pullPolicy: IfNotPresent
+  tag: v1.9.4@sha256:6d576b5aa9c08305026ff76ba9e08e47c40dfce958293f5b3307cae05cff873a
+
+service:
+  main:
+    ports:
+      main:
+        port: 8080
+        targetPort: 8080
+
+probes:
+  liveness:
+    enabled: true
+    ## Set this to true if you wish to specify your own livenessProbe
+    custom: true
+    ## The spec field contains the values for the default livenessProbe.
+    ## If you selected custom: true, this field holds the definition of the livenessProbe.
+    spec:
+      exec:
+        command:
+        - /usr/bin/env
+        - bash
+        - -c
+        - curl --fail localhost:8080
+      failureThreshold: 5
+      initialDelaySeconds: 60
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 10
```

Comment:
By default our probes, do TCP probing.
Or you can do something like this for http
```
probes:
  liveness:
    path: "/"
  readiness:
    path: "/"
  startup:
    path: "/"
```